{
  "info": {
    "author": "Mohd. Shafikur Rahman",
    "author_email": "shafikshaon@gmail.com",
    "bugtrack_url": null,
    "classifiers": [
      "Environment :: Web Environment",
      "Intended Audience :: Developers",
      "License :: OSI Approved :: MIT License",
      "Operating System :: OS Independent",
      "Programming Language :: Python :: 2.7",
      "Programming Language :: Python :: 3"
    ],
    "description": "# django-debug-helper\nDjango debug error related exception search on Google/ Stack Overflow.\n\n### Installation\n```\n$ pip install django-debug-helper\n```\n\n### Setting it up\nAdd the custom middleware to your **MIDDLEWARE** in the ```settings.py```.\n```\nif DEBUG:\n\tMIDDLEWARE += ('django-debug-helper.debug.ErrorSearchMiddleware', )\n```\nNext time you get an error in your applications, you will see an external link to a custom **google/ stackoverflow** search as you defined.\n\n\nBy default it set to **google** search, but if you want to use **stackoverflow** search instead of **google** then add this into your ```settings.py```\n```\nDJANGO_TRACE_SEARCH_SITE = \"stackoverflow\"\n```\n\n\n\n",
    "description_content_type": "text/markdown",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/shafikshaon/django-debug-helper",
    "keywords": "",
    "license": "MIT",
    "maintainer": "",
    "maintainer_email": "",
    "name": "django-debug-helper",
    "package_url": "https://pypi.org/project/django-debug-helper/",
    "platform": "",
    "project_url": "https://pypi.org/project/django-debug-helper/",
    "project_urls": {
      "Homepage": "https://github.com/shafikshaon/django-debug-helper"
    },
    "release_url": "https://pypi.org/project/django-debug-helper/1.1.3/",
    "requires_dist": null,
    "requires_python": "",
    "summary": "Django debug error related exception search on Google/ Stack Overflow.",
    "version": "1.1.3",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 4471849,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "57af80674863ecf5604855ae09c0a6430a48449eaabc742971c046996fd87823",
        "md5": "1c4fad17c3d7ef9d550a83cc27a66fed",
        "sha256": "42bbb88f23ab440e6c22e9132225644adcca49387f804c73cb40cf64dfcae50b"
      },
      "downloads": -1,
      "filename": "django_debug_helper-1.1.3-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "1c4fad17c3d7ef9d550a83cc27a66fed",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": null,
      "size": 4983,
      "upload_time": "2018-11-09T20:01:40",
      "upload_time_iso_8601": "2018-11-09T20:01:40.627159Z",
      "url": "https://files.pythonhosted.org/packages/57/af/80674863ecf5604855ae09c0a6430a48449eaabc742971c046996fd87823/django_debug_helper-1.1.3-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "0a330241a48bd3eee552a80cf22f5607f7ba7b1ff1c84f1a98a4d20191684cf7",
        "md5": "93a11600706f0c3f541fe436b991b925",
        "sha256": "e0e6d49ae11817b033a2175d4589f698ed75c6eafd5d8f16ffb18027deaf5ac9"
      },
      "downloads": -1,
      "filename": "django-debug-helper-1.1.3.tar.gz",
      "has_sig": false,
      "md5_digest": "93a11600706f0c3f541fe436b991b925",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": null,
      "size": 2487,
      "upload_time": "2018-11-09T20:01:43",
      "upload_time_iso_8601": "2018-11-09T20:01:43.963405Z",
      "url": "https://files.pythonhosted.org/packages/0a/33/0241a48bd3eee552a80cf22f5607f7ba7b1ff1c84f1a98a4d20191684cf7/django-debug-helper-1.1.3.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}